executors/cmdexec/block_volume.go

```diff
@@ -192,3 +194,163 @@ func (c *CmdExecutor) ListBlockVolumes(host string, blockhostingvolume string) (
 
 	return blockVolumeList.Blocks, nil
 }
+
+const (
+	GIGABYTE = 1 << (10 * iota)
+	TERABYTE
+	PETABYTE
+	EXABYTE
+)
+
+func toGigaBytes(s string) (int, error) {
```

Comment:
Since this function is critical to the proper functioning of this feature I must insist that we have some kind of unit test for this in this PR.